packages/taco/integration-test/encrypt-decrypt.test.ts

```diff
@@ -25,6 +25,9 @@ const DOMAIN = 'lynx';
 const RITUAL_ID = 27;
 const CHAIN_ID = 80002;
 
+// The wallet address of our consumer (derived from CONSUMER_PRIVATE_KEY)
+const CONSUMER_ADDRESS = '0x5aAeb6053F3E94C9b9A09f33669435E7Ef1BeAed';
```

Comment:
The public key of CONSUMER_PRIVATE_KEY (0xf307e1...) is different than this (correct one: 0xB0B3F5bf904aA4AeB770929BCFfC323439395878) if I'm not wrong.

In any case, I would derive the public key instead of hardcoding it to make easier code maintenance.

```suggestion
// The wallet address of our consumer
const CONSUMER_ADDRESS = ethers.utils.computeAddress(CONSUMER_PRIVATE_KEY);
```